flaky test "[node/child_process spawn] Verify that stdin and stdout work"

https://github.com/denoland/deno_std/pull/1228/checks?check_run_id=3581109307

```

error: Test failed

failures:

[node/child_process spawn] Verify that stdin and stdout work
AssertionError: Values are not strictly equal:


    [Diff] Actual / Expected


+   console.log("hello");
    


    at assertStrictEquals (file:///Users/runner/work/deno_std/deno_std/testing/asserts.ts:379:9)
    at fn (file:///Users/runner/work/deno_std/deno_std/node/child_process_test.ts:74:7)
    at async asyncOpSanitizer (deno:runtime/js/40_testing.js:35:9)
    at async resourceSanitizer (deno:runtime/js/40_testing.js:72:7)
    at async exitSanitizer (deno:runtime/js/40_testing.js:99:9)
    at async runTest (deno:runtime/js/40_testing.js:215:7)
    at async Object.runTests (deno:runtime/js/40_testing.js:283:22)
    at async file:///Users/runner/work/deno_std/deno_std/b622e01d-22de-40e7-8747-d3fafebe868d$deno$test.js:4:1

failures:

	[node/child_process spawn] Verify that stdin and stdout work

```

--- a/node/child_process_test.ts
+++ b/node/child_process_test.ts
@@ -67,7 +67,7 @@ Deno.test({
       });
       childProcess.stdin.write("  console.log('hello')", "utf-8");
       childProcess.stdin.end();
-      childProcess.on("exit", () => {
+      childProcess.on("close", () => {
         promise.resolve();
       });
       await promise;

